modules/runtime/scenario/scenario.py

```diff
@@ -72,7 +72,7 @@ def _build_world_state(self):
     for agent in self._agent_list:
       agent.GenerateRoadCorridor(self._map_interface)
       world.AddAgent(agent)
-    world.UpdateAgentRTree()
+      world.UpdateAgentRTree()
```

Comment:
why update it every single time? Please provide an explanation.

Reply:
I think this was unintended. Sorry for that.

Reply:
Please provide a fix for that.